examples/postprocess_dag.json

```diff
@@ -23,7 +23,10 @@
       "taxid_locations_family_nr.json",
       "taxid_locations_combined.json"
     ],
-    "alignment_viz_out": ["align_viz.summary"]
+    "alignment_viz_out": ["align_viz.summary"],
+    "srst2_out": ["output__genes__ARGannot_r2__results.txt"],
+    "fastqs": ["RR004_water_2_S23_R1_001.fastq.gz", "RR004_water_2_S23_R2_001.fastq.gz"],
+    "amr_db": ["ARGannot_r2.fasta"]
```

Comment:
As discussed, please move the reference database to additional_files when you get around to it.